testing/mailutils/APKBUILD

```diff
@@ -0,0 +1,74 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=mailutils
+pkgver=3.2
+pkgrel=0
+pkgdesc="GNU swiss army knife of electronic mail handling"
+url="https://mailutils.org/"
+arch="all"
+license="GPL3+"
+provides="mailx=$pkgver-r$pkgrel"
+replace="mailx"
+depends_dev="$pkgname-libs=$pkgver-r$pkgrel"
+makedepends="readline-dev libtool"
+checkdepends="autoconf"
+subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-servers $pkgname-mh"
+source="https://ftp.gnu.org/gnu/mailutils/mailutils-$pkgver.tar.gz
+	disable-koi8-r-test.patch"
+builddir="$srcdir/mailutils-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr
+	make -j4
+}
+
+check() {
+	cd "$builddir"
+	# exclude imap4d tests as they require chdir("/root") which won't work
+	sed -i 's/$(IMAP4D_DIR)/# $(IMAP4D_DIR)/g' Makefile
+	make check
+}
+
+package() {
+	cd "$builddir"
+	# re-enable imap4d subdirectory for install
+	sed -i 's/# $(IMAP4D_DIR)/$(IMAP4D_DIR)/g' Makefile
+	make DESTDIR="$pkgdir" install
+
+	rm "$pkgdir"/usr/lib/charset.alias
+	# no need for these to be suid/sgid root.
+	chmod u-s "$pkgdir"/usr/sbin/maidag
+	chmod g-s "$pkgdir"/usr/bin/dotlock
+}
+
+servers() {
+	pkgdesc="GNU swiss army knife of electronic mail handling (servers)"
```

Comment:
Will this actually work for multiple subpackages? I did it like this because I feared some subpackage may end up like "... (libraries) (servers)". :P